src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/GeneratorDiagnostics.cs

```diff
@@ -23,6 +23,8 @@ public class Ids
             public const string TypeNotSupported = Prefix + "1051";
             public const string ConfigurationNotSupported = Prefix + "1052";
             public const string CannotForwardToDllImport = Prefix + "1053";
+
+            public const string RequiresAllowUnsafeBlocks = Prefix + "1062";
```

Comment:
We need to update the `list-of-diagnostics.md` file to grab another diagnostic.

I might be able to get you one from the Analyzer that we currently aren't using and won't use with the new work.

Reply:
I was looking for a text file in the source generator directory but couldn't find one. I assumed in dotnet/runtime we generated them now. The name helped me to find it, I'll update. Thanks.